Eatery Blue/UI/HomeScreen/HomeModelController.swift

```diff
@@ -78,7 +78,11 @@ class HomeModelController: HomeViewController {
             allEateries = eateries.filter { eatery in
                 return !eatery.name.isEmpty
             }.sorted(by: { lhs, rhs in
-                lhs.name < rhs.name
+                if (lhs.isOpen == rhs.isOpen){
+                    lhs.name < rhs.name
+                } else {
+                    lhs.isOpen
+                }
```

Comment:
Next time, include in your PR description that if both eateries are open, sort by alphabetically order. Also, what if they are both closed? TBH I don't think alphabetically order matters if they're closed since nobody cares, but something to keep in mind and document.